### DISCUSSION TITLE
Translation task vs Transcribe English

### DISCUSSION THREAD

**Discussion by dodysw2** [openai/whisper#1685] [Q&A] [answered]
Given an audio containing language of Korean, what is the difference between:
* Running transcribe task with Language = en
* Running translate task with Language = ko

Both seems to give similar result.

**Comment by ryanheise** [comment, marked as answer]
Since the audio contains Korean, the language should always be set to Korean. Setting it to anything else is not defined and you shouldn't rely on the results.

Here are the valid inputs:

transcribe language=ko should give you the Korean transcript of your Korean audio, and
translate language=ko should give you the English translation of your Korean audio.
